tools/serverpod_cli/test/util/protocol_helper_test.dart

```diff
@@ -18,20 +18,23 @@ GeneratorConfig createGeneratorConfig([
     relativeDartClientPackagePathParts: [],
     modules: [],
     extraClasses: [],
+    serializeEnumValuesAsStrings: true,
   );
 }
 
+final serverRootDir = Directory(join(
+  'test',
+  'util',
+  'test_assets',
+  'protocol_helper',
+  'has_serverpod_server_project',
+  'test_server',
+));
+
+final config = createGeneratorConfig(split(serverRootDir.path));
```

Comment:
Suggestion: Move inside of test `main` method to scope their lifetime.

These variables are defined as soon as this files is included somewhere. Since assets created in a test should only be local to the tests the intention is more clear if we scope these to the main method of the test. We can still share them between the groups.